File: Java_MVP/app/src/main/java/com/gotoinc/requesin/repository/ApiContract.java
package com.gotoinc.requesin.repository;

import com.gotoinc.requesin.view.BaseViewContract;
import com.gotoinc.requesin.view.home.HomeContract;

import androidx.annotation.NonNull;

/**
 * Created by Illia Derevianko on 23.11.18.
 * GoTo Inc.
 */
public interface ApiContract {
    void getUsersList(int page, @NonNull HomeContract view, @NonNull BaseViewContract baseView);
}

File: Kotlin_MVP/app/src/main/java/com/gotoinc/requesin/repository/ApiContract.kt
package com.gotoinc.requesin.repository

import com.gotoinc.requesin.view.BaseViewContract
import com.gotoinc.requesin.view.home.HomeContract

/**
 * Created by Illia Derevianko on 28.11.18.
 * GoTo Inc.
 */
interface ApiContract {
    fun getUsersList(page: Int, view: HomeContract, baseView: BaseViewContract)
}